lib/generators/blacklight/blacklight_access_controls.rb

```diff
@@ -0,0 +1,19 @@
+Blacklight::AccessControls.configure do |config|
+  # This specifies the solr field names of permissions-related fields.
+  # The default fields used are shown below, if you index your permissions to other fields update the configuration below.
+  # If you change these, you must also update the permissions request handler in your solrconfig.xml to return those values
+  #
+  # config.permissions.discover.group       = Solrizer.solr_name("discover_access_group", :symbol)
+  # config.permissions.discover.individual  = Solrizer.solr_name("discover_access_person", :symbol)
+  # config.permissions.read.group           = Solrizer.solr_name("read_access_group", :symbol)
+  # config.permissions.read.individual      = Solrizer.solr_name("read_access_person", :symbol)
+  # config.permissions.edit.group           = Solrizer.solr_name("edit_access_group", :symbol)
+  # config.permissions.edit.individual      = Solrizer.solr_name("edit_access_person", :symbol)
+  #
+  # config.permissions.embargo.release_date  = Solrizer.solr_name("embargo_release_date", :stored_sortable, type: :date)
+  # config.permissions.lease.expiration_date = Solrizer.solr_name("lease_expiration_date", :stored_sortable, type: :date)
```

Comment:
You can remove the lease/embargo fields.  Those don't exist in blacklight-access_controls.